### PR TITLE
Capitalize first letter of lastName

### PR DESCRIPTION
**Pull request by Guillaume-Lazaro**
For faciliting alphabetical order

### CODE REVIEW

File: src/pages/edit-contact/edit-contact.ts [hunk 1]
```diff
@@ -72,6 +72,8 @@ export class EditContactPage {
 
   handleSubmit() {
     var toastMessage;
```
Comment (Ailelame) [inline src/pages/edit-contact/edit-contact.ts]: O.o' 
a var in typescript

Comment (Saryx11) [reply src/pages/edit-contact/edit-contact.ts]: technically, it's not a problem, but it's not good standards

**Review by Saryx11 — APPROVED** [review]

File: src/pages/edit-contact/edit-contact.ts (hunk 1 above)
Comment (Ailelame) [reply src/pages/edit-contact/edit-contact.ts]: NO! That's just bad, no var in typescript.

Comment (Guillaume-Lazaro) [reply src/pages/edit-contact/edit-contact.ts]: fixed

**Pull request merged**